src/pages/Creatures.tsx

```diff
@@ -160,13 +114,13 @@ export default function Creatures() {
                       : FIELDS[field].abbr || FIELDS[field].label
                   }
                   sort={query}
-                  reduceSort={reduceSort}
+                  reduceSort={filterProps.reduceSort}
```

Comment:
Lets still pass the specific props here so we don't couple the hook to the component too much.

You can make code so DRY it becomes brittle.